src/ontology/mondo.Makefile

```diff
@@ -266,19 +266,142 @@ $(TEMPLATES_DIR)/ROBOT_addMedGen_fromConflictResolution.tsv: tmp/July2023_CUIRep
 $(TEMPLATES_DIR)/ROBOT_addMedGen_fromIngest.tsv:
 	wget "https://github.com/monarch-initiative/medgen/releases/latest/download/medgen-xrefs.robot.template.tsv" -O $@
 
+## FIX THIS BEFORE MERGING
```

Comment:
@matentzn what is supposed to be fixed here?

Reply:
Oh yes. I doubled checked it, this has already been fixed on master branch, I deleted the goal (and comment)